app/assets/stylesheets/modules/star.scss

```diff
@@ -0,0 +1,18 @@
+.fa-paw {
+  color: #ffd700;
```

Comment:
Color literals like `#ffd700` should only be used in variable declarations; they should be referred to via variable everywhere else.